File: codigos-fonte/fibonacci.rs
//Fibonacci para Rust

struct elementos {
    a: i64, 
    b: i64, 
    ac: i64,
}

fn fibonacci(termos: usize){
    let mut valores = elementos {
        a: 0,
        b: 1,
        ac: 0,
    };
    for i in 0..termos{
        println!("{}", valores.a);
        valores.ac = valores.a + valores.b;
        valores.a = valores.b;
        valores.b = valores.ac;
    }
}

fn main(){
    fibonacci(64)
}

File: codigos-fonte/fibonacci.java
//Fibonacci para Java 

public class fibonacci {

	public class elementos {
		long a;
		long b;
		long ac;
	}

	public static int fibonacci(int termos) {
		a = 0;
		b = 1;
		ac = 0;
		for (int i = 1; i <= termos; i++) {
			System.out.println(a);
			ac = a + b;
			a = b;
			b = ac;
		}
		return 0;
	}

	public static void main(String args[]) {
		fibonacci(64);
	}
}
